src/main/kotlin/uk/gov/dluhc/printapi/mapper/EroManagementApiEroDtoMapper.kt

```diff
@@ -1,10 +1,48 @@
 package uk.gov.dluhc.printapi.mapper
 
 import org.mapstruct.Mapper
+import org.mapstruct.Mapping
 import uk.gov.dluhc.eromanagementapi.models.ElectoralRegistrationOfficeResponse
+import uk.gov.dluhc.printapi.dto.AddressDto
+import uk.gov.dluhc.printapi.dto.EroContactDetailsDto
 import uk.gov.dluhc.printapi.dto.EroManagementApiEroDto
 
 @Mapper
-interface EroManagementApiEroDtoMapper {
-    fun toEroManagementApiEroDto(eroResponse: ElectoralRegistrationOfficeResponse): EroManagementApiEroDto
+abstract class EroManagementApiEroDtoMapper {
+
+    @Mapping(target = "englishContactDetails", expression = "java( englishContactDetails() )")
+    @Mapping(target = "welshContactDetails", expression = "java( welshContactDetails() )")
+    abstract fun toEroManagementApiEroDto(eroResponse: ElectoralRegistrationOfficeResponse): EroManagementApiEroDto
+
+    // TODO - to be replaced with mapping from ElectoralRegistrationOfficeResponse when ERO Management returns contact data
+    protected fun englishContactDetails(): EroContactDetailsDto =
+        EroContactDetailsDto(
+            name = "Gwynedd Council Elections",
+            phoneNumber = "01766 771000",
+            website = "https://www.gwynedd.llyw.cymru/en/Council/Contact-us/Contact-us.aspx",
+            emailAddress = "TrethCyngor@gwynedd.llyw.cymru",
+            address = AddressDto(
+                property = "Gwynedd Council Headquarters",
+                street = "Shirehall Street",
+                town = "Caernarfon",
+                area = "Gwynedd",
+                postcode = "LL55 1SH",
+            )
+        )
+
+    // TODO - to be replaced with mapping from ElectoralRegistrationOfficeResponse when ERO Management returns contact data
+    protected fun welshContactDetails(): EroContactDetailsDto =
+        EroContactDetailsDto(
+            name = "Etholiadau Cyngor Gwynedd",
+            phoneNumber = "01766 771000",
+            website = "https://www.gwynedd.llyw.cymru/cy/Cyngor/Cysylltu-%c3%a2-ni/Cysylltu-%c3%a2-ni.aspx",
```

Comment:
These are the Welsh details for Gwynedd Council, including accented characters in their website URL and street 👍

Reply:
Interesting, so we'd expect the URL to be stored and provided with URL encoding - not really a question for this PR - just pondering aloud.

Reply:
Yeah, I don't know TBH. It might be that they only provide the link to the "home page" , which in this case would be `https://www.gwynedd.llyw.cymru/cy/Cyngor` ?
I've used a link here however to a page on their site (their contact us page) as it has "special" characters in the path.
I'm assuming here that they copy and paste the URL encoded version (which is what I did), but the version that appears in the browser bar has the "special" characters in it:
<img width="1142" alt="Screenshot 2022-11-15 at 09 31 49" src="https://user-images.githubusercontent.com/78348259/201883378-6c30869f-b7df-428c-9a02-aa4cf43f32b9.png">